auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/GrpcUmsClient.java

```diff
@@ -789,19 +789,20 @@ public String getIdentityProviderMetadataXml(String accountId) {
         return resourceRoleAssignments;
     }
 
-    public void assignResourceRole(
-            String assigneeCrn, String resourceCrn, String resourceRoleCrn, Optional<String> requestId) {
+    public void assignResourceRole(String assigneeCrn, String resourceCrn, String resourceRoleCrn, Optional<String> requestId) {
         UmsClient client = makeClient(channelWrapper.getChannel());
-        LOGGER.info("Assigning {} role for resource {} to assignee {}", resourceRoleCrn, resourceCrn, assigneeCrn);
+        LOGGER.info("Assigning '{}' role at resource '{}' for assignee '{}'...", resourceRoleCrn, resourceCrn, assigneeCrn);
         client.assignResourceRole(RequestIdUtil.getOrGenerate(requestId), assigneeCrn, resourceCrn, resourceRoleCrn);
-        LOGGER.info("Assigned {} role for resource {} to assignee {}", resourceRoleCrn, resourceCrn, assigneeCrn);
+        LOGGER.info("Resource role assignment has been successfully done: \nrole: {} \nresource: {} \nassignee: {}",
```

Comment:
I am not sure `\n` works in kibana

Reply:
https://github.com/elastic/kibana/issues/76242  is not yet implemented, please remove them